packages/trader/src/Modules/SmartChart/Components/Markers/accumulators-profit-loss-text.tsx

```diff
@@ -1,19 +1,18 @@
 import React from 'react';
 import { Text } from '@deriv/components';
-import { formatMoney, getCurrencyDisplayCode, isMobile } from '@deriv/shared';
+import { addComma, formatMoney, getCurrencyDisplayCode, isMobile } from '@deriv/shared';
 import { FastMarker } from 'Modules/SmartChart';
 import { FastMarkerBeta } from 'Modules/SmartChartBeta';
 import classNames from 'classnames';
 import { TRef } from './accumulators-profit-loss-tooltip';
 import { ProposalOpenContract } from '@deriv/api-types';
 
-type TProposalOpenContractProfit = Required<Pick<ProposalOpenContract, 'profit'>>;
-
-type TAccumulatorsProfitLossText = Pick<ProposalOpenContract, 'current_spot' | 'current_spot_time' | 'currency'> &
-    TProposalOpenContractProfit & {
-        className?: string;
-        is_beta_chart?: boolean;
-    };
+type TAccumulatorsProfitLossText = Pick<ProposalOpenContract, 'current_spot' | 'current_spot_time' | 'currency'> & {
+    className?: string;
+    displayed_profit_value: number;
```

Comment:
`displayed` normally refers to a string value that doesn't need any formatting changes but here it refers to a raw value. Can the name be changed?

Reply:
i'll make it `profit_value`